app/views/user_mailer/account_reset_cancel.html.slim

```diff
@@ -1,16 +0,0 @@
-p.lead == t('.intro', app: link_to(APP_NAME, AppConfig.env.mailer_domain_name, class: 'gray'))
-
-table.spacer
-  tbody
-    tr
-      td.s10 height="10px"
-        | &nbsp;
-table.hr
-  tr
-    th
-      | &nbsp;
-
-p == t('.help',
-        app: link_to(APP_NAME, AppConfig.env.mailer_domain_name, class: 'gray'),
-        help_link: link_to(t('user_mailer.help_link_text'), MarketingSite.help_url),
-        contact_link: link_to(t('user_mailer.contact_link_text'), MarketingSite.contact_url))
```

Comment:
this link didn't end up in the SLIM, it's one of the blank strings right?

Reply:
yep: https://github.com/18F/identity-idp/pull/4477/commits/3e6efb54674514faff66ca637753ba388f9c5122#diff-baed13852b4e017692745c34e348c3275753faebb04220447834c17731f9f46cL11